WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletInfoViewModel.cs

```diff
@@ -0,0 +1,65 @@
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Userfacing;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Advanced
+{
+	[NavigationMetaData(Title = "Wallet Info")]
+	public partial class WalletInfoViewModel : RoutableViewModel
+	{
+		[AutoNotify] private bool _showSensitiveData;
+		[AutoNotify] private string _showButtonText = "Show sensitive data";
+		[AutoNotify] private string _lockIconString = "eye_show_regular";
+
+		public WalletInfoViewModel(Wallet wallet)
+		{
+			var network = wallet.Network;
+
+			SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+			EnableCancel = !wallet.KeyManager.IsWatchOnly;
```

Comment:
why?

Reply:
There is no sensitive data on HW and WatchOnly wallets.
Here it doesn't behave as a cancel button. Detailed answer here https://github.com/zkSNACKs/WalletWasabi/pull/5823#discussion_r636000748